applications/mobu/README.md

```diff
@@ -13,7 +13,9 @@ Continuous integration testing
 | affinity | object | `{}` | Affinity rules for the mobu frontend pod |
 | config.autostart | list | `[]` | Autostart specification. Must be a list of mobu flock specifications. Each flock listed will be automatically started when mobu is started. |
 | config.debug | bool | `false` | If set to true, include the output from all flocks in the main mobu log and disable structured JSON logging. |
-| config.githubIntegration | bool | `true` | Whether to enable the GitHub app integration. This should be `false` in envs that are behind a VPN because the integration depends on being able to receive webhook requests from github.com. |
+| config.github | object | `{}` | GitHub app integration options. |
```

Comment:
Put an @default in values.yaml for this so we don't have an inlined object; sure, since in this case it's empty it's no big deal, but it's good to get into the habit.